tests/test_keys.py

```diff
@@ -197,6 +197,29 @@ def test_format_metadata_to_key(self):
 
 
 
+  def test_format_metadata_to_key_non_default_hash_algorithms(self):
+    # Reconfiguring rsakey_dict to conform to KEY_SCHEMA
+    # i.e. {keytype: 'rsa', keyval: {public: pub_key, private: priv_key}}
+    keyid = self.rsakey_dict['keyid']
```

Comment:
As a matter of taste I would much prefer that, rather than saving the value of the keyid here and restoring it later, we just make a copy of `rsakey_dict` to use within the test. Then we don't have to worry about restoring the class member to its pristine state at the end of the test.